.changeset/angry-ants-reply.md

```diff
@@ -0,0 +1,82 @@
+---
+'@twilio-paste/codemods': patch
+'@twilio-paste/modal': patch
+'@twilio-paste/core': patch
+'@twilio-paste/website': patch
+'@twilio-paste/color-contrast-utils': patch
+'@twilio-paste/alert': patch
+'@twilio-paste/alert-dialog': patch
+'@twilio-paste/anchor': patch
+'@twilio-paste/avatar': patch
+'@twilio-paste/badge': patch
+'@twilio-paste/base-radio-checkbox': patch
+'@twilio-paste/breadcrumb': patch
+'@twilio-paste/button': patch
+'@twilio-paste/card': patch
+'@twilio-paste/checkbox': patch
+'@twilio-paste/combobox': patch
+'@twilio-paste/data-grid': patch
+'@twilio-paste/date-picker': patch
+'@twilio-paste/disclosure': patch
+'@twilio-paste/display-pill-group': patch
+'@twilio-paste/form-pill-group': patch
+'@twilio-paste/heading': patch
+'@twilio-paste/help-text': patch
+'@twilio-paste/inline-control-group': patch
+'@twilio-paste/input': patch
+'@twilio-paste/input-box': patch
+'@twilio-paste/label': patch
+'@twilio-paste/list': patch
+'@twilio-paste/menu': patch
+'@twilio-paste/pagination': patch
+'@twilio-paste/paragraph': patch
+'@twilio-paste/popover': patch
+'@twilio-paste/radio-group': patch
+'@twilio-paste/screen-reader-only': patch
+'@twilio-paste/select': patch
+'@twilio-paste/separator': patch
+'@twilio-paste/skeleton-loader': patch
+'@twilio-paste/spinner': patch
+'@twilio-paste/table': patch
+'@twilio-paste/tabs': patch
+'@twilio-paste/textarea': patch
+'@twilio-paste/time-picker': patch
+'@twilio-paste/toast': patch
+'@twilio-paste/tooltip': patch
+'@twilio-paste/truncate': patch
+'@twilio-paste/aspect-ratio': patch
+'@twilio-paste/flex': patch
+'@twilio-paste/grid': patch
+'@twilio-paste/media-object': patch
+'@twilio-paste/stack': patch
+'@twilio-paste/box': patch
+'@twilio-paste/combobox-primitive': patch
+'@twilio-paste/disclosure-primitive': patch
+'@twilio-paste/menu-primitive': patch
+'@twilio-paste/modal-dialog-primitive': patch
+'@twilio-paste/non-modal-dialog-primitive': patch
+'@twilio-paste/sibling-box': patch
+'@twilio-paste/tabs-primitive': patch
+'@twilio-paste/text': patch
+'@twilio-paste/tooltip-primitive': patch
+'@twilio-paste/cra-template': patch
+'@twilio-paste/customization': patch
+'@twilio-paste/design-tokens': patch
+'@twilio-paste/icons': patch
+'@twilio-paste/animation-library': patch
+'@twilio-paste/clipboard-copy-library': patch
+'@twilio-paste/data-visualization-library': patch
+'@twilio-paste/dropdown-library': patch
+'@twilio-paste/reakit-library': patch
+'@twilio-paste/styling-library': patch
+'@twilio-paste/uid-library': patch
+'@twilio-paste/nextjs-template': patch
+'@twilio-paste/style-props': patch
+'@twilio-paste/theme': patch
+'@twilio-paste/theme-designer': patch
+'@twilio-paste/token-contrast-checker': patch
+'@twilio-paste/types': patch
+'@twilio-paste/utils': patch
+---
+
+esbuild dependency update for compiled source code
```

Comment:
question: what does this mean?

Reply:
suggestion: "Updated a build dependency (esbuild) which changes the output of our builds slightly, without materially changing anything about the code."

Reply:
Yeah, what he said. Becuase we change the compiled code _ever so slightly_ with this esbuild upgrade, we need to force publish all packages even though nothing actually changes